extensions/keycloak/deployment/src/main/java/io/quarkus/keycloak/KeycloakConfig.java

```diff
@@ -11,53 +11,53 @@
 import io.quarkus.runtime.annotations.ConfigRoot;
 import io.quarkus.runtime.annotations.DefaultConverter;
 
-@ConfigRoot(name = "keycloak")
+@ConfigRoot
 public final class KeycloakConfig {
 
     /**
      * Name of the realm.
      */
-    @ConfigItem(name = "realm")
+    @ConfigItem
     String realm;
 
     /**
-     * Name of the realm.
+     * Name of the realm key.
      */
     @ConfigItem(name = "realm-public-key")
     Optional<String> realmKey;
```

Comment:
Let's also try to be consistent here. Let's rename the property and remove the `name`.

Reply:
BTW, I'm talking about renaming the *Java* property.

Reply:
Yes. While I was at it I renamed several others config items.